go/vt/vttablet/tabletmanager/replication_reporter.go

```diff
@@ -113,6 +118,9 @@ func repairReplication(ctx context.Context, agent *ActionAgent) error {
 	if !si.HasMaster() {
 		return fmt.Errorf("no master tablet for shard %v/%v", tablet.Keyspace, tablet.Shard)
 	}
+	if *mysqlctl.DisableActiveReparents {
```

Comment:
This is unreachable now, isn't it?

Reply:
Ah yes good point. I will add a guard clause instead, because if for some reason we in the future by mistake call this method we should get an error instead.